gsd-schema/osv-gsd/validation/schema-osv-gsd-1.0.1.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://json-schema.org/draft/2020-12/schema",
-  "$id": "https://schema.gsd.id/gsd.id/osv-gsd/schema-osv-gsd-1.0.0.json",
+  "$id": "https://schema.gsd.id/gsd.id/osv-gsd/1.0.1/schema.json",
```

Comment:
Forgot to also update the osv_gsd_schema_version enum, just need to make that match 1.0.1